tools/qmctl/qmctl

```diff
@@ -278,14 +318,15 @@ def main():
         description="QMCTL: Manage QM container settings via Podman.",
         epilog="""
 Examples:
-  ./qmctl show                             Show raw container config
-  ./qmctl show resources                   Show live systemd-cgtop for qm.service
-  ./qmctl show unix-domain-sockets        Show UNIX sockets inside container
-  ./qmctl show shared-memory              Show shared memory (ipcs)
-  ./qmctl show namespaces                 Show namespaces from container
-  ./qmctl show available-devices          Show devices from container config
-  ./qmctl show all                        Show all options + cgtop snapshot
-  ./qmctl exec uname -a                   Run 'uname -a' in container
+  ./qmctl show                              Show raw container config
```

Comment:
why it keep suggestion change here? New line issue maybe?

Reply:
I aligned it to be in the same line. Should I change it back?

Reply:
No, all good than.